### ISSUE TITLE
[a11y] 9.3.2.4 - Consistent labelling

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5395]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------
## General notes
Responsive (Mobile) view: when the menue (Hamburger-Button) is closed the label of the button is "Navigationsmenü  ffnen" and when the user wants to close the menue the label of the button is "Seitenleiste schließen".
This could lead to irritations for the users. Also the area on the right of the page "Seitenleiste" appears as label in the close button "X".

## Status: partly fulfilled

**Comment by pascalwengerter** [comment]
The first part I get and it's super straightforward to fix (literally changing one string). The second part I can't grok

**Comment by kulmann** [comment]
I looked into it and what I found is that the hamburger button in larger resolutions still shows `Navigationsmenü öffnen` despite the sidebar already being open. Currently fixing it. Needs a new store module though.

**Comment by kulmann** [comment]
Ah, that was unrelated. Found out what was meant by looking into the german version 😅 The term `Seitenleiste` is already in use for the right sidebar. Apparently we're using the term for both sidebars. Needs to be more strict.

**Comment by kulmann** [comment]
Fixed in https://github.com/owncloud/web/pull/5512/commits/893bc29f217720aecc922580b4fce71b992d18bd

**Comment by pascalwengerter** [comment]
Just checked and can't reproduce any of the mentioned issues (app switcher and sidebar have been heavily reworked in the meantime..)
